bin/osg-notify

```diff
@@ -199,12 +205,16 @@ def main():
 
             if verify_send.lower() == 'y':
                 session = smtplib.SMTP('localhost')
-                session.sendmail("help@opensciencegrid.org", recipients, msg)
+                for recipient, msg in zip(recipients, msgs):
+                    session.sendmail("help@opensciencegrid.org", recipient, msg)
+                    time.sleep(4)
```

Comment:
Put a `print(".", end="")` in the for loop to let the user know that something is happening.